multi/multi_test.go

```diff
@@ -204,3 +207,57 @@ func TestRefresh(t *testing.T) {
 		t.Error("Expect to get data after reconnect")
 	}
 }
+
+// runTestMain is a body of TestMain function
+// (see https://pkg.go.dev/testing#hdr-Main).
+// Using defer + os.Exit is not works so TestMain body
+// is a separate function, see
+// https://stackoverflow.com/questions/27629380/how-to-exit-a-go-program-honoring-deferred-calls
+func runTestMain(m *testing.M) int {
+	var err error
```

Comment:
It looks like you can replace it with `:=` (for `err` and `inst1, inst2`)

Reply:
Well, it worked. I expected go linter to fail because I'm trying to set a type to already existing var `err` while creating second instance.